src/main/asciidoc/query-by-example.adoc

```diff
@@ -186,3 +191,28 @@ The following table describes the scope of the various `ExampleMatcher` settings
 | Property path
 
 |===
+
+[[query-by-example.fluent]]
+== Fluent API
+
+`QueryByExampleExecutor` offers one more method, which we did not mention so far: `<S extends T, R> R findBy(Example<S> example, Function<FluentQuery.FetchableFluentQuery<S>, R> queryFunction)`.
+As with other methods it executes a query derived from an `Example`.
+But with the second argument you can control aspects of that execution, that you can't control dynamically otherwise.
+You do so by invoking the various methods of the `FetchableFluentQuery` in the second argument.
+`sortBy` allows you to specify an ordering for your result.
+`as` allows you to specify the type to which you want the result to be transformed.
+`project` limits the attributes queried.
+`first`, `firstValue`, `one`, `oneValue`, `all`, `page`, `stream`, `count`, `exists` define what kind of result you'll get and also how the query behaves when more than the expected number of results are available.
```

Comment:
Change to:

`first`, `firstValue`, `one`, `oneValue`, `all`, `page`, `stream`, `count`, and `exists` define what kind of result you get and how the query behaves when more than the expected number of results are available.